mods/ra2/rules/defaults.yaml

```diff
@@ -392,6 +392,7 @@
 		Voice: Attack
 	ActorLostNotification:
 	BodyOrientation:
+		QuantizedFacings: 32
```

Comment:
I think the point is there is no reason to limit it to 32 when it's actually a voxel and doesn't care about the number of facings. 32 is perfectly fine for sprite actors to look good, so limiting a voxel actor to 32 will leave it OK-looking, but there is still no reason to limit it.